src/confetti.js

```diff
@@ -308,6 +308,30 @@
     };
   }
 
+  function drawStar(context, cx, cy, spikes, outerRadius, innerRadius){
+    context.save();
+    
+    var rot=Math.PI/2*3;
```

Comment:
I am not sure what `rot` is. Do you mean rotation? I think we can probably afford the extra characters